docs/JestObjectAPI.md

```diff
@@ -119,6 +119,92 @@ test('implementation created by jest.genMockFromModule', () => {
 });
 ```
 
+This is how `genMockFromModule` will mock the follwing data types:
+
+#### `Function`
+
+A new function will be created. The new function will have no formal parameters and when called will return `undefined`. This functionality also applies to `async functions`.
+
+#### `Class`
+
+A new class will be created. The interface of the original class is maintained however all of the class member functions will be mocked.
```

Comment:
What about class properties, if there's a property that is an array will it be mocked with the array strategy mentioned below?

Reply:
Good point, I can add this case to the example code / description.